lib/providers/DplCmsConfigContextProvider.tsx

```diff
@@ -3,17 +3,32 @@
 import { createContext } from "react"
 
 import { TDplCmsPublicConfig } from "../config/dpl-cms/configSchemas"
+import { useGetDplCmsPublicConfigurationQuery } from "../graphql/generated/dpl-cms/graphql"
 
 export const DplCmsConfigContext = createContext<TDplCmsPublicConfig | null>(null)
 
-export default function DplCmsConfigContextProvider({
-  children,
-  config,
-}: {
-  children: React.ReactNode
-  config: TDplCmsPublicConfig | null
-}) {
+export default function DplCmsConfigContextProvider({ children }: { children: React.ReactNode }) {
+  const { data } = useGetDplCmsPublicConfigurationQuery()
+  const goConfiguration = data?.goConfiguration?.public
+    ? {
+        loginUrls: {
+          adgangsplatformen: data.goConfiguration.public.loginUrls?.adgangsplatformen ?? null,
+        },
+        logoutUrls: {
+          adgangsplatformen: data.goConfiguration.public.logoutUrls?.adgangsplatformen ?? null,
+        },
+        libraryInfo: {
+          name: data.goConfiguration.public.libraryInfo?.name ?? null,
+        },
+        unilogin: {
+          municipalityId: data.goConfiguration.public.unilogin?.municipalityId ?? null,
+        },
+      }
+    : null
```

Comment:
You are replicating what is happening [here](https://github.com/danskernesdigitalebibliotek/dpl-go/blob/325bd14fa1c0a114af3c60abd50ea4828d7c6715/lib/config/dpl-cms/dplCmsConfig.ts#L98) you should use `getDplCmsPublicConfigData()` instead

Reply:
I think also you need to handle suspense on the preview page. One Cypress test is failing...